cloudsmith_cli/core/api/packages.py

```diff
@@ -255,7 +255,7 @@ def get_parameters(cls):
         for k, v in six.iteritems(cls.swagger_types):
             attr = getattr(cls, k)
             docs = attr.__doc__.strip().split("\n")
-            doc = (docs[1] if docs[1] else docs[0]).strip()
+            doc = (docs[2].lstrip() if docs[2] else docs[0]).strip()
```

Comment:
NAB: Would it be better to use a list comprehension here (or a generator expression?) which ignores blank strings?

Reply:
Something like:

```python
docs = [s.strip() for s in attr.__doc__.strip().split("\n") if s.strip()]
doc = ...
```

Reply:
Good idea - done 👍